buildpacks/jvm/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for new OpenJDK versions: `24.0.0` ([#788](https://github.com/heroku/buildpacks-jvm/pull/788))
+
+### Changed
+
+- Default version for **OpenJDK 24** is now `24.0.0` ([#788](https://github.com/heroku/buildpacks-jvm/pull/788))
```

Comment:
Should this line be omitted for new major version releases? There is no change in default version for OpenJDK version 24, since it wasn't previously supported :-)

Reply:
Yeah, I removed it :)